__tests__/api.test.js

```diff
@@ -219,3 +219,39 @@ describe("/POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/PATCH /api/articles/:article_id", () => {
+  const updateVotes = {
```

Comment:
Never leave test variables outside the test blocks. This object is vulnerable to mutation by all tests as it is in the scope of all tests. Redeclare this inside each test block.

Reply:
Noted! I have implemented your feedback to be included my next pull request task 11. thank you!